package.json

```diff
@@ -26,25 +28,41 @@
   },
   "homepage": "https://zen-audio-player.github.io",
   "devDependencies": {
+    "eslint": "1.10.x",
     "htmlhint": "0.9.x",
-    "jshint": "^2.9.1-rc1",
     "mocha": "2.3.x",
     "zombie": "4.x"
   },
   "engines": {
     "node": ">=4.x"
   },
-  "jshintConfig": {
-    "curly": true,
-    "camelcase": true,
-    "indent": 4,
-    "loopfunc": true,
-    "newcap": true,
-    "node": true,
-    "quotmark": "double",
-    "trailing": true,
-    "unused": true,
-    "white": true,
-    "esversion": 6
+  "eslintConfig": {
+    "extends": "eslint:recommended",
+    "env": {
+        "browser": true,
+        "mocha": true,
+        "jquery": true,
+        "es6": true
+    },
+    "globals": {
+        "YT": true,
+        "ga": true,
+        "twttr": true,
+        "time": true,
+        "require": true,
+        "__dirname": true,
```

Comment:
Are `__dirname` and `__filename` required in this list? Can you add `"node": true` to the `env` object?